web/modules/custom/sfgov_profiles/sfgov_profiles.info.yml

```
@@ -0,0 +1,5 @@
+name: 'SFGov Profiles'
+type: module
+description: 'SF Gov Profiles'
+core: 8.x
```

Lets make this D9 ready :)
```suggestion
core_version_requirement: ^8.8 || ^9
```